.github/workflows/deploy.yml

```diff
@@ -0,0 +1,42 @@
+name: Deploy
+
+on:
+  push:
+    tags:
+    - '*'
+
+jobs:
+  carthage:
+    name: Upload Carthage binary 
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Select Xcode 11.4.1
+        run: sudo xcode-select -switch /Applications/Xcode_11.4.1.app
+      - uses: actions/cache@v1
+        with:
+          path: Carthage
+          key: ${{ runner.os }}-carthage-${{ hashFiles('**/Cartfile.resolved') }}
+          restore-keys: |
+            ${{ runner.os }}-carthage-
+      - name: Build
+        run: carthage build --no-skip-current --cache-builds
+      - name: Archive
+        run: carthage archive ACKategories
+      - name: Upload artifact
+        uses: fnkr/github-action-ghr@v1
+        if: startsWith(github.ref, 'refs/tags/')
+        env:
+          GHR_PATH: ACKategories.framework.zip
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  cocoapods:
+    name: Push podspec to Cocoapods trunk
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Select Xcode 11.4.1
+        run: sudo xcode-select -switch /Applications/Xcode_11.4.1.app
```

Comment:
I wonder if we can extract this into a variable since the version is used in multiple places

Reply:
I had this idea too, but haven't done any research for this...do you have any experience with this? Just first solution that comes to my mind would be adding a file to repository and then reading it in action 🤷‍♂️

Reply:
I was thinking more about this and the obvious solution would be to have some shared storage for all workflows. 

The most simple solution would be to have something like _secrets_ but one that would be visible to everyone (or we can use secrets as they are 🤷‍♂️). The real drawback of this solution would be, that in this case the Xcode version will not be versioned and we could end in cases that you might have a stale branch which uses some older version of Xcode and it will become uncompilable.

So we need to have a solution that would save Xcode version to repository itself. As we would need to read the file with Xcode version several times in various workflows and jobs it will be handy to create GitHub action for this.

Reply:
Great job with the custom action, I love it 🥳